resolwe_bio/processes/jbrowse/variants.yml

```diff
@@ -0,0 +1,39 @@
+# =========================
+# JBrowse tracks - Variants
+# =========================
+#
+---
+
+- slug: jbrowse-vcf-track
```

Comment:
I don't like the way we support BAMs, but it does save disk space.
Having a special case for VCF too would mean making another additional request to the server (or a new custom endpoint), instead of being clear `query(type_startswith = data:jbrowse:)`